Sources/AWSLambdaRuntimeCore/HTTPClient.swift

```diff
@@ -131,10 +138,10 @@ internal final class HTTPClient {
     }
 
     internal struct Response: Equatable {
-        public var version: HTTPVersion
-        public var status: HTTPResponseStatus
-        public var headers: HTTPHeaders
-        public var body: ByteBuffer?
+        var version: HTTPVersion
+        var status: HTTPResponseStatus
+        var headers: HTTPHeaders
+        var body: ByteBuffer?
```

Comment:
This is a cleanup change. `public` access control in a `internal struct` doesn't make much sense.